SPDX-FileCopyrightText: 2023 fosslinux <fosslinux@aussies.space>

SPDX-License-Identifier: GPL-3.0-or-later

In new gettext external is required for AM_GNU_GETTEXT.

--- binutils-2.30/intl/configure.ac	2023-02-07 18:57:56.350832016 +1100
+++ binutils-2.30/intl/configure.ac	2023-02-07 18:58:07.310054484 +1100
@@ -4,7 +4,7 @@
 AC_CONFIG_HEADER(config.h)
 AC_CONFIG_MACRO_DIR(../config)
 AM_GNU_GETTEXT_VERSION(0.12.1)
-AM_GNU_GETTEXT([], [need-ngettext])
+AM_GNU_GETTEXT([external], [need-ngettext])
 
 # This replaces the extensive use of DEFS in the original Makefile.in.
 AC_DEFINE(IN_LIBINTL, 1, [Define because this is libintl.])
